.github/workflows/ci_eval.yaml

```diff
@@ -7,6 +7,7 @@
 name: CI - Perplexity
 
 on:
+  pull_request:
```

Comment:
I assume we don't want to run this on every PR as it takes quite some time?

Reply:
Yes, this is only added to debug.